specification/databricks/resource-manager/readme.python.md

```diff
@@ -4,7 +4,7 @@ These settings apply only when `--python` is specified on the command line.
 Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
 
 ``` yaml $(python)
-title: DatabricksClient
+title: AzureDatabricksManagementClient
```

Comment:
@scottmacchia-msft  `azure-mgmt-databricks` has been GAed so the change will cause breaking change for customers. Please revert this line